test/command_callback/test_eclipselsp_command_callback.vader

```diff
@@ -0,0 +1,87 @@
+Before:
+  call ale#assert#SetUpLinterTest('java', 'eclipselsp')
+  let b:ale_java_eclipselsp_path = '/home/user/eclipse.dst.ls'
+  call ale#test#SetFilename('dummy.java')
+
+  let b:cfg = '/home/user/eclipse.dst.ls/org.eclipse.jdt.ls.product/target/repository/config_linux'
+
+  if has('win32')
+    let b:cfg = '\home\user\eclipse.dst.ls\org.eclipse.jdt.ls.product\target\repository\config_win',
+  elseif has('macunix')
+    let b:cfg = '/home/user/eclipse.dst.ls/org.eclipse.jdt.ls.product/target/repository/config_mac',
```

Comment:
The commas here break the tests on Windows and Mac. If you remove those, AppVeyor should pass.

Reply:
Fixed.